apps/wallet/src/ui/app/components/WalletListSelectItem.tsx

```diff
@@ -76,30 +81,43 @@ export function WalletListSelectItem({
             clearTimeout(timeout);
         };
     }, [isNew]);
+
+    const isActiveModeDisconnect = mode === SELECTION_MODE.DISCONNECT;
```

Comment:
I would change those to `isDisconnect` and `isSelect` just to be smaller but fine as is (I was happy with the way it was currently done as well! 😄 )